src/main/java/com/amazon/opendistroforelasticsearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -185,6 +186,22 @@ public PrivilegesEvaluatorResponse evaluate(final User user, String action0, fin
             log.debug("action: "+action0+" ("+request.getClass().getSimpleName()+")");
         }
 
+        if (request instanceof BulkRequest && (Strings.isNullOrEmpty(user.getRequestedTenant()))) {
```

Comment:
String?

Reply:
Fixed the import. "Strings" class was not imported.